bears/general/LineCountBear.py

```diff
@@ -18,13 +18,14 @@ def _get_blank_line_count(self, file):
         return num_blank_lines
 
     def run(self, filename, file, max_lines_per_file: int,
-            exclude_blank_lines: bool=False):
+            exclude_blank_lines: bool=False, min_lines_per_file: int=0):
         """
-        Count the number of lines in a file and ensure that they are
-        smaller than a given size.
+        Count the number of lines in a file and ensure that they lie
+        within the range of given sizes.
 
-        :param max_lines_per_file: Number of lines allowed per file.
+        :param max_lines_per_file: Maximum number of lines allowed per file.
         :param exclude_blank_lines: "True" if blank lines are to be excluded.
+        :param min_lines_per_file: Minimum number of lines allowed per file.
         """
         file_length = len(file)
         if exclude_blank_lines:
```

Comment:
Maybe leave a line before and after every `if` block to improve code readability. :)